lang/funcs/operator_polyfunc.go

```diff
@@ -39,6 +39,11 @@ const (
 )
 
 func init() {
+	epsilon := func() float64 {
```

Comment:
I bet you can do even better! this computation still repeats each time.

Reply:
updated!